### DISCUSSION TITLE
Fundamentos do AspNet 6 - Aula: Token e JWT

### DISCUSSION THREAD

**Discussion by danilobicas-unoeste** [balta-io/forum#570] [Tira Dúvidas] [answered]
Olá Balta.
No caso do uso do token para autorização e autenticação e já imaginando que no meu controller eu tenho a injeção do meu serviço de domínio (private readonly IServico _servico;) que conecta com o BD... como eu faço para que o usuário que veio do token seja o mesmo que vai na conexão com o BD (para que os log's do meu banco sejam feitos com o usuário conectado no momento)? É assim que usa? ou o usuário da conexão do BD em uma API é sempre o mesmo?

Obrigado

Atte.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @danilobicas-unoeste, como vai?

Não sei se entendi muito bem, mas acredito que seja isto:

```csharp
[HttpPost("v1/products")]
public async Task<IResult> PostAsync(Request request)
{
    var user = User.Identity.Name; // Obtém usuário logado
    request.UserId = user; // Passa para o request
    ...
}
```
